Synthetics gRPC - mention support for unary only (#15014)
DataDog/documentation: content/en/synthetics/api_tests/grpc_tests.md

2. Upload a [`.proto` file][1] that defines your gRPC server.

   - Select the service and method you want to send a gRPC message to from the dropdown menu.  

>    We currently do not support streaming methods. These methods are greyed out in-app.

Reviewer: ```suggestion
     Datadog does not support streaming methods, which are greyed out in-app.
```
